images/singleuser-sample/Dockerfile

```diff
@@ -6,21 +7,15 @@ FROM python:3.9-bullseye as build-stage
 
 WORKDIR /build-stage
 
-# Build wheels for packages that require gcc or other build dependencies and
-# lack wheels either for amd64 or aarch64.
-#
-#   - https://pypi.org/project/psutil/#files, no wheels available as of 5.9.1
-#     for aarch64. See https://github.com/giampaolo/psutil/pull/2070.
-#
-# If you find a dependency here no longer listed in requirements.txt, remove it
-# from here as well. The more wheels we build here and copy and then install,
-# where the copy and install are separate steps, the larger the final image
-# becomes.
-#
+# set env for pip cache,
+# but use ARG to avoid persisting in image
```

Comment:
```suggestion
# set pip's cache directory using this environment variable, and use
# ARG instead of ENV to ensure its only set when the image is built
```

Reply:
Spot on!

Probably doesn't matter and we could use ENV. I don't know what the best practice is for environment variables that you want to use in a Dockerfile, but don't necessarily want to have in the final image.